modules/serverless-event-delivery-component-behaviors.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// serverless/event_workflows/serverless-event-delivery.adoc
+
+[id="serverless-event-delivery-component-behaviors_{context}"]
+= Event delivery behavior for Knative Eventing components
+
+Different Knative Eventing components have their own behavior patterns that are followed for event delivery.
+
+== Delivery failure status codes
+
+The channel or broker receiver may respond with the following status codes if an event fails to be delivered:
```

Comment:
That's not true as the component can also e.g. retutn `200` to indicate success. This is only the list of the error codes, not a complete list of all status codes.

Reply:
It's not supposed to be a comprehensive list, I just added these because they were the ones @slinkydeveloper had on hand / gave me at the time when I asked if there were other codes. As mentioned above, please add a new issue if you feel we need a full reference doc for status codes for this, since it doesn't really fall into the scope of the original issue / PR, which is to provide config info for event delivery. We mention the code that means it was successful under the other sections, this section as the heading mentions is specifically related to showing _some_ possible failure codes.

Reply:
I indeed think such a list (or better table) of all status codes would be very helpful as a reference. Should this be opened as GitHub issue or as JIRA ticket ? 

I mean the use case is typical, that you get a status code and want to know the reason for this. How could I resolve this without having a comprehensive mapping of all codes?

Again, I don't want to derail the PR here, so happy to continue the discussion somewhere else.

Reply:
@rhuss as for the others, please open it as a Jira issue.